rails-tasks/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>note to self<br>must do:</h1>
+<table align="center">
+  <% @tasks.each do |task| %>
+    <tr>
+      <td>
+        <% if task.complete == true %>
+          <a id=complete href="/task_list/<%= task.name %>"> <%= task.name %> </a>
+        <% else %>
+          <a id=incomplete href="/task_list/<%= task.name %>"> <%= task.name %> </a>
```

Comment:
I think this will result in having mutliple `a` tags with the same ID which is not good